Detectors/TPC/simulation/include/TPCSimulation/DigitContainer.h

```diff
@@ -80,17 +80,23 @@ class DigitContainer
   size_t size() const { return mTimeBins.size(); }
 
  private:
-  TimeBin mFirstTimeBin = 0;            ///< First time bin to consider
-  TimeBin mEffectiveTimeBin = 0;        ///< Effective time bin of that digit
-  TimeBin mTmaxTriggered = 0;           ///< Maximum time bin in case of triggered mode (hard cut at average drift speed with additional margin)
-  TimeBin mOffset = 700;                ///< Size of the container for one event
-  std::deque<DigitTime> mTimeBins{700}; ///< Time bin Container for the ADC value
+  TimeBin mFirstTimeBin = 0;       ///< First time bin to consider
+  TimeBin mEffectiveTimeBin = 0;   ///< Effective time bin of that digit
+  TimeBin mTmaxTriggered = 0;      ///< Maximum time bin in case of triggered mode (hard cut at average drift speed with additional margin)
+  TimeBin mOffset;                 ///< Size of the container for one event
+  std::deque<DigitTime> mTimeBins; ///< Time bin Container for the ADC value
```

Comment:
Why not simply keep the default initialization?

Reply:
It is anyways set in the constructor so why should we keep a default here?

Reply:
Ok, should not really matter, indeed.